shimmingtoolbox/coils/coil.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+
+import numpy as np
+from typing import Tuple
+
+
+class Coil(object):
+    """
+    Coil profile object that stores coil profiles and there constraints
+
+    Attributes:
+        dim (Tuple[int]): Dimension along specific axis. dim: 0,1,2 are spatial axes, while dim: 3 corresponds to the
+                          coil channel.
+        profile (np.ndarray): (dim1, dim2, dim3, channels) 4d array of N 3d coil profiles
+        affine (np.ndarray): 4x4 array containing the qform affine transformation for the coil profiles
+        coef_sum_max (float): Contains the maximum value for the sum of the coefficients
+        coef_channel_minmax (list): Contains the maximum coefficient for each channel
```

Comment:
Isn't it weird to call it "minmax" if it contains the maximum coefficient ?

Reply:
Yes.. I wanted to find a way to include the "max" negative current/coefficient